proxy/agent.go

```diff
@@ -78,6 +78,12 @@ var (
 	}
 )
 
+const (
+	// MaxAborts is the maximum number of cascading abort messages to buffer.
+	// This should be the upper bound on the number of proxies available at any point in time.
+	MaxAborts = 10
```

Comment:
there might be a race in how fast envoy kills old envoys. so 3 is approximate (I have never seen more than 3).